src/lib/serviceRoutinesV2/getEntities.cpp

```diff
@@ -70,6 +70,11 @@ std::string getEntities
   string    idPattern  = ciP->uriParam["idPattern"];
   string    q          = ciP->uriParam["q"];
 
+  if (ciP->uriParam["options"] == "count")
```

Comment:
About "Good enough for now" ...
That statement comes from before ciP->uriParamOptions was implemented.

But ... now it is implemented. That changes everything, right?

And of course, we need to fix those older 'good enough for then' to use uriParamOptions. It is not good enough any more.

I just added this to issue #1041